### ISSUE TITLE
Requesting a clear and comprehensive list of "dev-sounding" subdomains

### ISSUE THREAD

**Issue by lindseydiloreto** [craftcms/docs#192]
In the knowledgebase, there is a page regarding License Enforcement. On it is an important section regarding ["How do we determine Craft is running on a public domain?"](https://craftcms.com/knowledge-base/how-craft-license-enforcement-works#how-do-we-determine-craft-is-running-on-a-public-domain)

<img width="898" alt="How do we determine Craft is running on a public domain?" src="https://user-images.githubusercontent.com/5309692/113629537-6522e080-961b-11eb-9460-6b5b20c514d7.png">

There has always been some confusion about the **"dev-sounding subdomain"** (buried on Step 4). For starters, these subdomain choices could probably stand to be be highlighted a little more, or given their own separate space.

More importantly, people often struggle with what matches the description of a "dev-sounding subdomain". Is the list provided above comprehensive? Or are these really just examples ("e.g.") representing a subset of what's possible?

Excluding the number of times the question has been asked on Slack/Discord, here are a few documented examples of people asking for clarification on Stack Exchange...

 - https://craftcms.stackexchange.com/questions/4417/craft-license-what-is-considered-a-dev-sounding-subdomain
 - https://craftcms.stackexchange.com/questions/37510/does-preprod-count-as-a-dev-sounding-sub-domain
 - https://craftcms.stackexchange.com/questions/8790/getting-admin-to-work-on-a-dev-sounding-subdomain

Perhaps we just need a formal list of acceptable subdomains, to answer the question once and for all.

**Comment by mattstein** [comment]
We unearthed the complete Step 4 list from the Vault of Secrets, where there were only a handful of additional dev-sounding subdomains. (And they’re not very exciting.)

Updated [the page](https://craftcms.com/knowledge-base/how-craft-license-enforcement-works#how-do-we-determine-craft-is-running-on-a-public-domain) to _exhaustively_ list those subdomains and added a link to the comprehensive list of known development domains. 

That should vanquish any ambiguity once and for all as you suggest. Thanks for the issue!
